app/views/shared/_profile-accordion.html.erb

```diff
@@ -1,6 +1,6 @@
 <section class="accordion">
 
-    <button class="accordion__control" aria-expanded="true">
+    <button class="accordion__control" aria-expanded="false">
```

Comment:
Why did you change this? This block is expanded by default, so I think this should remain `true`, no?

Reply:
I agree - this shouldn't change.